src/components/ProjetoDialog.tsx

```
@@ -213,27 +237,43 @@ export function ProjetoDialog({
                 <ArrowLeft className="h-4 w-4" />
                 Voltar
               </Button>
-              {/*
-                    
-              {/* Botão Candidatar-se (apenas para alunos) }
-              {AuthService.isAluno() && (
+
+              {/* Candidatar-se (se ALUNO ou SERVIDOR, e não participa nem já candidatou) */}
+              {(roles?.includes('ROLE_ALUNO') || roles?.includes('ROLE_SERVIDOR')) &&
```

ESLint

[error] Insert `⏎···············`

(prettier/prettier)